LEAF_Nexus/auth_domain/index.php

```diff
@@ -18,82 +16,92 @@
 
 $login = new Orgchart\Login($db, $db);
 
-if(isset($_SERVER['REMOTE_USER'])) {
-	$protocol = 'http://';
-	if(isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') {
-		$protocol = 'https://';
-	}
-	$redirect = '';
-	if(isset($_GET['r'])) {
+if (isset($_SERVER['REMOTE_USER']) && Orgchart\Config::$leafSecure == false)
```

Comment:
Consider that some sites may not have the new config variable $leafSecure. Check if the variable exists before accessing it, or create a config update script to update old config files.

Reply:
created script `./updateToLEAFSecure.sh` in commit `efa` that adds variable to configs